tests/009-test-follow-symlinked-devices.sh

```diff
@@ -0,0 +1,50 @@
+source $SRCDIR/libtest.sh
+
+test_follow_symlinked_devices() {
+  local devs dev
+  local test_status=1
+  local testname=`basename "$0"`
+  local vg_name="dss-test-foo"
+
+  # Create a symlink for a device and try to follow it
+  for dev in $TEST_DEVS; do
+    devlink="$dev-test.$$"
+    ln -s $dev $devlink
+    devs="$devs $devlink"
+    echo "Using symlinked devices: $devlink -> $(readlink -e $devlink)" >> $LOGS 
+  done
+
+  cat << EOF > /etc/sysconfig/docker-storage-setup
+DEVS="$devs"
+VG=$vg_name
+EOF
+  # Run docker-storage-setup
+  $DSSBIN >> $LOGS 2>&1
+
+  # Test failed.
+  if [ $? -ne 0 ]; then
+    cleanup $vg_name "$TEST_DEVS"
+    for dev in $devs; do
+      if [ -h $dev ];then
```

Comment:
We don't require this `-h` test again. We know all $dev are soft links as we have created these just now.